doc/man3/flux_msg_handler_create.adoc

```diff
@@ -30,6 +32,12 @@ SYNOPSIS
 
  void flux_msg_handler_stop (flux_msg_handler_t *mh);
 
+ void flux_msg_handler_allow_rolemask (flux_msg_handler_t *w,
```

Comment:
Both of these functions really need a clear explanation of how they work.  It is a little unual, I think, to have a function that or's the supplied mask rather than just setting the mask, and another that and's the inverse of the supplied mask.  I think simple "set" and "get" mask functions would have been better.  But since this is what we have, we should document what they do.